imports/plugins/core/versions/server/migrations/25_deafult_parcel_size.js

```diff
@@ -0,0 +1,25 @@
+import { Migrations } from "meteor/percolate:migrations";
+import { Shops } from "/lib/collections";
+import { convertWeight, convertLength } from "/lib/api";
+
+Migrations.add({
+  version: 25,
+  up() {
+    const shops = Shops.find().fetch();
+    shops.forEach((shop) => {
+      const defaultParcelSize = {
+        weight: convertWeight("lb", shop.baseUOM, 8),
+        height: convertLength("in", shop.baseUOL, 6),
+        length: convertLength("in", shop.baseUOL, 11.25),
+        width: convertLength("in", shop.baseUOL, 8.75)
+      };
+      Shops.update({ _id: shop._id }, { $set: { defaultParcelSize } });
```

Comment:
Add a third argument: `{ bypassCollection2: true }`

This prevents the migration from failing as the schema changes in the future. For example, if `defaultParcelSize` is someday removed from the shop schema, we still want this to be able to set it without throwing an error because a later migration would remove it.